src/BlockStorage/v2/Models/Volume.php

```diff
@@ -156,4 +159,42 @@ public function parseMetadata(ResponseInterface $response): array
 
         return isset($json['metadata']) ? $json['metadata'] : [];
     }
+
+    /**
+     * Update the bootable status for a volume, mark it as a bootable volume.
+     *
+     * @param bool $bootable
+     */
+    public function setBootable(bool $bootable)
```

Comment:
Seems like this should have a default value (true). The name of the method insinuates that it makes the volume bootable, rather than necessarily setting a flag.